modules/utilityFunction.js

```diff
@@ -0,0 +1,127 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export function loadBookslist(bookInstanceFromBookStore) {
+  function loopingBook(book) {
+    return `<div id="${book.id}" class="book-card">
```

Comment:
- [ ] The keyword `function` is depreciated in the new javascript ES6 syntax and is replaced by the arrow functions. So I would suggest you to use  `arrow functions` instead of function keyword as well. 

Example snippet:

```js

export const loadBookslist = (bookInstanceFromBookStore) => {
...
}

```